Packages/manifest.json

```diff
@@ -13,14 +13,14 @@
     "com.unity.netcode.gameobjects": "1.4.0",
     "com.unity.performance.profile-analyzer": "1.2.2",
     "com.unity.postprocessing": "3.2.2",
-    "com.unity.render-pipelines.universal": "14.0.7",
-    "com.unity.services.authentication": "2.5.0",
+    "com.unity.render-pipelines.universal": "14.0.8",
+    "com.unity.services.authentication": "2.7.1",
     "com.unity.services.lobby": "1.1.0-pre.3",
     "com.unity.services.relay": "1.0.5",
-    "com.unity.services.wire": "1.1.5",
+    "com.unity.services.wire": "1.2.0",
```

Comment:
After having a closer look, we can actually remove this from the manifest, and simply keep wire as a dependency of the Lobby package. Could do it while we are here, or later in the lobby update PR if that makes more sense

Reply:
Since you have a PR open for that Lobby update, I'll let you make those changes inside that PR.